### PR TITLE
Make all docs references relative

### PR DESCRIPTION
**Pull request by aforward**
For supporting GH markdown and GH (web)pages

We can ensure the links work in GH with
https://github.com/eclipse-pass/main/tree/relative-links-everywhere

And we cannot fully test on eclipse-pass.org/main until it's deployed :-(

### CODE REVIEW

File: docs/infra/README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The following describes the current PASS Infrastructure.
 
 EC2 virtual machines provide access to internal resources within the AWS VPC.
 
-Learn more about [deploying pass-docker via EC2](/docs/infra/ec2.md).
+Learn more about [deploying pass-docker via EC2](infra/ec2.md).
```
Comment (aforward) [inline docs/infra/README.md:15]: fix me

**Review by bbranan — APPROVED** [review]
Looks good in the standard github view. I don't see a way to test the links via the GitHub Pages hosting, so I'll just assume this works as expected there.

**Comment by aforward** [comment]
Correct, we just need to deploy and try.

**Comment by bbranan** [comment]
Perfect, links now work as expected in the GH Pages view: https://eclipse-pass.org/main/. Thanks @aforward!

**Pull request merged**